src/Voron/Data/Tables/IndexDefinitions.cs

```diff
@@ -0,0 +1,443 @@
+using System;
+using System.Diagnostics;
+using System.Reflection;
+using Sparrow;
+using Sparrow.Binary;
+using Sparrow.Server;
+
+namespace Voron.Data.Tables
+{
+    public unsafe partial class TableSchema
```

Comment:
Given that this is a partial class let's rename the file to `TableSchema.IndexDefinitions.cs` to follow the convention that we have.

Reply:
done